spec/helpers/index.ts

```diff
@@ -1,22 +1,36 @@
+import { State, Listing } from '../../src/interfaces'
 import store from '../../src/redux/store'
 import { applyListing } from '../../src/redux/action-creators/registry'
-
+import { getListings } from '../../src/redux/selectors'
 
 const createListing = async (userAddress: string) => {
+  let state: State = store.getState()
+  let originalListings: Listing[] = getListings(state)
+  let hashes: string[] = originalListings.map((item) => item.listingHash)
+
   // generate a random listing value
   const listing: string = 'listing ' + Math.floor(Math.random() * 1000 * 1000).toString()
```

Comment:
shouldn't this be interpolated rather than concat'd:  ... `listing ${Math.floor(...)`